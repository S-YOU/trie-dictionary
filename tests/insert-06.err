nodes: 2
nodes: 3
nodes: 4
nodes: 5
nodes: 6
nodes: 7
nodes: 8
nodes: 9
nodes: 10
nodes: 11
