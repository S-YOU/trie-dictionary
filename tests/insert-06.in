insert gwghrsejeiz
insert gw
insert gwghrsejeizsmzz
insert gwghrsej
insert gwghrs
insert gwg
insert gwghrseje
insert gwghrsej
insert gwghrs
insert gwghrsej
insert gwg
insert g
insert gwghrsejeizs
insert gw
insert gwghrsejeizsmzz
insert gwghrsejeizsmz
