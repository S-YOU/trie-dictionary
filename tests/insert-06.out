word number: 0
word number: 1
word number: 2
word number: 3
word number: 4
word number: 5
word number: 6
ignored
ignored
ignored
ignored
word number: 7
word number: 8
ignored
ignored
word number: 9
